### PR TITLE
Add note about `http-server.https.automatic-shared-secret parameter`

### PR DESCRIPTION
**Pull request by radek-starburst**
## Description
Add note about `http-server.https.automatic-shared-secret` parameter

## Release notes

() This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

### CODE REVIEW

File: docs/src/main/sphinx/security/internal-communication.md [hunk 1]
```diff
@@ -108,6 +108,10 @@ configuration properties:
 - `internal-communication.https.keystore.path`
 - `internal-communication.https.truststore.path`
 - `node.internal-address-source`
+
+Also, note that you need to configure `http-server.https.automatic-shared-secret`
```
Comment (mosabua) [inline docs/src/main/sphinx/security/internal-communication.md:112]: Why not add this to point 4? Or 2 .. or both? .. does this really belong in this note?

**Review by electrum — CHANGES_REQUESTED** [review]
I'm confused about the purpose of this change. This property is set automatically in `InternalCommunicationModule` as appropriate. It should not be configured manually.

Also, the note is in the section that lists the old properties to be removed. Why would that section list a property that needs to be added?

**Comment by radek-starburst** [comment]
> This property is set automatically in InternalCommunicationModule as appropriate. It should not be configured manually.

I followed documentation https://trino.io/docs/current/security/internal-communication.html step by step to enable secure internal communication. When I was running Trino it was failing because of parameter `http-server.https.automatic-shared-secret` was not set. This is why I decided to add note about this parameter.

**Comment by mosabua** [comment]
What was the outcome of you testing @radek-starburst ?

**Comment by radek-starburst** [comment]
It was my mistake.

> I assume everything is working without this change...

yes

**Pull request closed without merging**